src/game_inventory.cpp

```diff
@@ -2769,3 +2769,158 @@ std::pair<item_location, bool> game_menus::inv::unload( Character &you )
 
     return inv_s.execute();
 }
+
+class reload_inventory_preset : public inventory_selector_preset
+{
+    public:
+        reload_inventory_preset( Character &you, const item_location &target, bool empty ) : you( you ),
+            target( target ), empty( empty ) {
+            _indent_entries = false;
```

Comment:
```suggestion
            _indent_entries = false;
            _collate_entries = true;
```
Seems like a good candidate for collation. https://github.com/CleverRaven/Cataclysm-DDA/pull/63270#issuecomment-1402391091

Reply:
Yeah, I can see how that'd be useful.
![grafik](https://github.com/CleverRaven/Cataclysm-DDA/assets/38702195/8375599d-33b3-4cbd-a473-f5b3233e5821)
Slightly exaggerated case, but otherwise every ammo box would be its own entry, with 3 clips per entry.